h2o-docs/src/product/data-science/glm.rst

```diff
@@ -1341,6 +1341,223 @@ You can extract the columns in the Coefficients Table by specifying ``names``, `
+
+**Poisson**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i) = w_i \big(y_i \times \log (\mu_i) - \mu_i - \log (\Gamma (y_i +1)) \big)
+
+where
+
+- :math:`\mu_i` is the prediction
+- :math:`y_i` is the real value of the target variable
+
+**Negative Binomial**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i, w_i) = y_i \log \big(\frac{k \mu}{w_i} \big) - \big(y_i + \frac{w_i}{k} \big) \log \big(1 + \frac{k \mu}{w_i} \big) + \log \Big(\frac{\Gamma \big( y_i = \frac{w_i}{k} \big)} {\Gamma (y_i +1) \Gamma \big(\frac{w_i}{k}\big)} \Big)
+
+where
+
+- :math:`\mu_i` is the prediction
+- :math:`y_i` is the real value of the target variable
+- :math:`k = \frac{1}{\phi}` is the dispersion parameter estimation
+
+.. note::
+   
+   You must estimate the dispersion parameter. During initialization, ``compute_p_values`` and ``remove_collinear_columns`` must be set to ``True`` to facilitate this estimation process. Also, ``dispersion_parameter_method`` must be set to ``"ml"``.
+
+   Since ``dispersion_parameter_method="ml"`` only works without regularization, you disable regularization by setting ``lambda`` equal to an array of ``0.0``.
+
+**Gamma**:
+
+.. math::
+   
+   l(\mu_i (\beta); y_i, w_i) = \frac{w_i}{\phi} \log \big( \frac{w_i y_i}{\phi \mu_i} \big) - \frac{w_i y_i}{\phi \mu_i} - \log (y_i) - \log \big(\Gamma \big(\frac{w)i}{\phi} \big) \big)
+
+where
+
+- :math:`\mu_i` is the prediction
+- :math:`y_i` is the real value of the target variable
+- :math:`\phi` is the dispersion parameter estimation
+
+.. note::
+   
+   You must estimate the dispersion parameter. During initialization, ``compute_p_values`` and ``remove_collinear_columns`` must be set to ``True`` to facilitate this estimation process. Also, ``dispersion_parameter_method`` must be set to ``"ml"``.
+
+   Since ``dispersion_parameter_method="ml"`` only works without regularization, you disable regularization by setting ``lambda`` equal to an array of ``0.0``.
+
+**Multinomial**:
+
+.. math::
+   
+   l(\mu_i(\beta); y_i) = w_i \log (\mu_i)
+
+where :math:`\mu_i` is the predicted probability of the actual class :math:`y_i`
+
+**Tweedie**:
+
+The Tweedie calculation is located in the section `Tweedie Likelihood Calculation <#tweedie-likelihood-calculation>`__.
+
+.. note::
+   
+   You must estimate the dispersion parameter. During initialization, ``compute_p_values`` and ``remove_collinear_columns`` must be set to ``True`` to facilitate this estimation process. Also, ``dispersion_parameter_method`` must be set to ``"ml"``.
+
+   Since ``dispersion_parameter_method="ml"`` only works without regularization, you disable regularization by setting ``lambda`` equal to an array of ``0.0``.
+
+Final AIC Calculation
+^^^^^^^^^^^^^^^^^^^^^
+
+The final AIC in the output metric is calculated using the standard formula, utilizing the previously computed log likelihood.
+
+.. math::
+   
+   \text{AIC} = -2LL + 2p
+
+where
+
+- :math:`p` is the number of parameters estimated in the model
```

Comment:
Please change 

:math:`p` is the number of parameters estimated in the model

to

:math:`p` is the number of non-zero coefficients estimated in the model